### PR TITLE
[integration-tests] Set VSX_REGISTRY_URL to fix workspace tests

### PR DESCRIPTION
**Pull request by WVerlaek**
## Description

Fix workspace integration tests, see https://gitpod.slack.com/archives/C01KGM9BH54/p1667868092950329

## How to test

Run integration tests

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-integration-tests=workspace
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Review by jenting — APPROVED** [review]
LGTM

/hold to wait CI pass

**Comment by WVerlaek** [comment]
Initial run of integration tests failed to create some workspaces in `TestGitHubContexts/context`, because workspaces from the same context were already running, even though `force-new` was specified in the CreateWorkspace request 🤔. 

(in the failed tests, the `CreateWorkspace` request returned a workspace in the `ExistingWorkspaces` field of the response)

**Comment by WVerlaek** [comment]
Looks like `force-new` and other modes got removed from the API in https://github.com/gitpod-io/gitpod/pull/14461/files#diff-957d9dd15b947e3969539ebd0dabcb8b40ce8b5564e5820cd9bb27d88be1dbc7L1388 earlier today

cc @svenefftinge I see there's some new fields added to replace the mode field, but don't see them in the [`CreateWorkspaceOptions` go struct](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/go/gitpod-service.go#L1988) yet, does this still need to be updated?

**Comment by svenefftinge** [comment]
PR: https://github.com/gitpod-io/gitpod/pull/14528

File: test/pkg/integration/workspace.go [hunk 1]
```diff
@@ -179,6 +179,14 @@ func LaunchWorkspaceDirectly(t *testing.T, ctx context.Context, api *ComponentAP
 				Email:    "integration-test@gitpod.io",
 			},
 			Admission: wsmanapi.AdmissionLevel_ADMIT_OWNER_ONLY,
+			Envvars: []*wsmanapi.EnvironmentVariable{
+				// VSX_REGISTRY_URL is set by server, since we start the workspace directly
+				// from ws-manager in these tests we need to set it here ourselves.
+				{
+					Name:  "VSX_REGISTRY_URL",
+					Value: "http://open-vsx.gitpod.io/",
```
Comment (akosyakov) [inline test/pkg/integration/workspace.go:187]: https I believe

Comment (WVerlaek) [reply test/pkg/integration/workspace.go:187]: looked like this worked, but see https also used elsewhere, will update

Comment (akosyakov) [reply test/pkg/integration/workspace.go:187]: ok

**Pull request closed without merging**